MaxiOps/CCTP_Bridge/receive/claimBaseUSDC.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "8443",
```

Comment:
this is incorrect and breaking ci report

Reply:
report actually flagged this already -_-

Reply:
<img width="772" alt="Screenshot 2024-05-09 at 16 34 28" src="https://github.com/BalancerMaxis/multisig-ops/assets/2835259/fbee16b1-a8f2-4f60-ab40-4f31d2e4b04e">